src/snowflakes/renderers.py

```diff
@@ -226,13 +227,13 @@ def should_transform(request, response):
         if request.authorization is not None:
             format = 'json'
         else:
-            mime_type = request.accept.best_match(
+            acceptable = request.accept.acceptable_offers(
```

Comment:
This is a pyramid 1.10 update?

Reply:
WebOb update (Pyramid's request/response object.) Silencing a deprecation warning on the best_match method. https://docs.pylonsproject.org/projects/webob/en/stable/api/webob.html#webob.acceptparse.AcceptValidHeader.best_match